src/redist/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    },
+    "csc": "1.0.0-featmsbuild-*",
+    "dotnet": "1.0.0-featmsbuild-*",
+    "nuget": "1.0.0-featmsbuild-*",    
+    "tool_msbuild": "1.0.0-featmsbuild-*"
```

Comment:
Is it guaranteed implicitly?

IIRC we've run into issues where packages were being pulled where we expected projects. Being explicit here is better IMO.